rcl/test/rcl/test_get_actual_qos.cpp

```diff
@@ -430,10 +430,7 @@ get_parameters(bool for_publisher)
       });
     }
   } else {
-    if (rmw_implementation_str == "rmw_connext_cpp" ||
-      rmw_implementation_str == "rmw_connext_dynamic_cpp" ||
-      rmw_implementation_str == "rmw_opensplice_cpp")
-    {
+    if (rmw_implementation_str == "rmw_opensplice_cpp") {
```

Comment:
We can just remove this whole block of code; we don't support opensplice anymore.

Reply:
I removed this reference to opensplice in [eca1b0c](https://github.com/ros2/rcl/pull/895/commits/eca1b0c1fdaa6131b55bd957d76a1b667051a1fc)